### PR TITLE
174 set realistic parameters for paper example

### PR DESCRIPTION
**Pull request by eriktamsen**
Updated the input parameter for the optimization workflow to be as close to the final values as can be estimated right now.
Some small additions to the snakemake workflow.

### CODE REVIEW

File: lebedigital/demonstrator_scripts/computation_sc_volume_fraction.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def computation_sc_volume_fraction(input_dic):
+    # initialize output dictionary
+    output = {}
+
+    if input_dic["sc_mass_fraction"] == 0:
+        return 0
+    else:
+        mass_sub = 1 * ureg("kg") / (1 + (1 - input_dic["sc_mass_fraction"]) / input_dic["sc_mass_fraction"])
```
Comment (joergfunger) [inline lebedigital/demonstrator_scripts/computation_sc_volume_fraction.py:13]: not sure I can follow this formula, is that the mass of cement substitute per m³ of concrete?

File: lebedigital/demonstrator_scripts/computation_sc_volume_fraction.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        print(f"mass_sub = {mass_sub}")
+        mass_cem = 1 * ureg("kg") - mass_sub
```
Comment (joergfunger) [inline lebedigital/demonstrator_scripts/computation_sc_volume_fraction.py:15]: m_c = 1kg-m_s?? see my comments in the other review

**Review by joergfunger — CHANGES_REQUESTED** [review]
I think I somehow reviewed the wrong merge request, but the comments are pretty much the same? Should I redo these comments, or could you work on the other merge request?

**Comment by eriktamsen** [comment]
@joergfunger: First of all, thanks for going through it. I will check your questions one by one. Quite possible that with all the additional changes, some mistakes have been introduced. I will try to make it work with the comments in the other PR. Should I answer them in the other branch and add changes to this one? Or should we just work with the other PR. Basically this branch is already merged in the other one.

**Comment by joergfunger** [comment]
If everything is in the other branch already, (and that was that one that I accidently reviewed), I would suggest using the other and deleting this one.

**Comment by eriktamsen** [comment]
Closing this PR, as already merged in newer PR #181

**Pull request closed without merging**